Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
 # L5: Principal Software Engineer
 
-> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
+> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus product and technical strategies, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
 
-- **Planning horizon**: 1+ year
-- **Impact radius**: Group+ (15-50). Note: This role is evolving as the company grows. PSE is right at the top of our IC track. For example, the impact radius expectations of an L5 will likely grow as Octopus grows larger over the next couple years, from 2-4 teams in 2021. We expect the role to stabilise in 2022 once the company has grown to the point of needing an L6 IC engineering role.
-- **Evaluation**: Director or VP
-- **Responsibility and direction needed**: Drives technical strategy in alignment with product strategy and drives high-impact improvements across teams.
+- **Thinking horizon**: 1 year
+- **Impact radius**: Group
+- **Evaluation**: Director
+- **Responsibility and direction needed**: I drive technical strategy in alignment with product strategy for my group. Point me in the right direction with helpful constraints and good outcomes happen with minimal oversight.
```

Comment:
```suggestion
- **Responsibility and direction needed**: I drive technical strategy in alignment with product strategy for my group.
```

The second sentence is unlikely to add information as equivalent sentiment also appears in the L4 definition (as `Consistently takes brand new ideas from inception to production with limited supervision`).